cmd/get/launch_plan_util.go

```diff
@@ -49,10 +50,11 @@ func FetchAllVerOfLP(ctx context.Context, lpName string, project string, domain
 			Name:    lpName,
 		},
 		SortBy: &admin.Sort{
-			Key:       "created_at",
+			Key:       config.GetConfig().SortBy,
```

Comment:
I don't know if you do this in other places but I generally recommend the class/function to take in the config as a parameter to make unit testing easier... and in the "real" invocation, you just do config.GetConfig() for it...